test/Constellation/TestUtil.hs

```diff
@@ -27,6 +27,7 @@ setupTestNode :: FilePath -> String -> IO (TVar Node, Int)
 setupTestNode d name = do
     kp1@(pub1, _) <- newKeyPair
     kp2@(pub2, _) <- newKeyPair
+    selfPub       <- fst <$> newKeyPair
```

Comment:
tiny nit: using `fst` when the previous two lines pattern match

Reply:
Fixed